agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1217,7 +1232,7 @@ bool backhaul_manager::send_autoconfig_search_message(std::shared_ptr<sRadioInfo
         LOG(ERROR) << "addClass ieee1905_1::tlvAlMacAddress failed";
         return false;
     }
-    tlvf::mac_from_string(tlvAlMacAddress->mac().oct, bridge_info.mac);
+    tlvAlMacAddress->mac() = db->bridge.mac;
```

Comment:
```suggestion
    auto bridge_mac = db->bridge.mac;
    tlvAlMacAddress->mac() = bridge_mac;
```
as you use it several times in this scope

Reply:
It is struct access.